README.md

```diff
@@ -323,7 +323,8 @@ down --way --the --all --my-custom-flag
 
 The rough heuristic here is if we have custom arguments:
 
-1. If a `builder <action>` command, append with ` -- ` to pass through.
+1. If a `builder <action>` command, pass through using builder-specific
```

Comment:
yeah, I read through that section of the docs a few times, and I guess I'm just asking for more specific language. Just so I've got this right, the supported  cases are like: 
1. `builder <cmd> -- --flag`
2. `builder <cmd> <json-array>`
3. `<non--builder cmd> --flag`

Is that right? After reading the docs, I dont find it 100% clear what `builder-specific environment variables. (Builder uses `_BUILDER_ARGS_CUSTOM_FLAGS`).` means. I can open a separate docs clarity issue though, and I'm happy to take it on so long as I can bug you with questions like this.